src/auth_middleware.rs

```diff
@@ -169,10 +169,10 @@ describe! auth_middleware_tests {
         let db = UsersDb::new();
         db.clear().ok();
         let user = UserBuilder::new()
-            .id(1).name("username")
-            .password("password")
-            .email("username@example.com")
-            .secret("secret")
+            .id(1).name(String::from("username"))
```

Comment:
After discussing in the #rust channel of IRC, this is the preferred form to create `String` values. I changed the user builder to accept `String` in place of `&str` following motivations explained below.